core/state_transition.go

```diff
@@ -591,6 +601,9 @@ func (st *stateTransition) innerExecute() (*ExecutionResult, error) {
 			ReturnData: ret,
 		}, nil
 	}
+
+	st.refundOperatorCost()
```

Comment:
Let's flank this call by `if rules.IsOptimismIsthmus { ... }`, then we can remove the more complex check in `refundOperatorCost`

Reply:
Done